plugin/util/parse_bindings.go

```diff
@@ -88,7 +88,7 @@ func parseBindingObjList(topList *ast.ObjectList) (map[string]StringSet, error)
 
 		resourceList := item.Val.(*ast.ObjectType).List
 		for _, rolesItem := range resourceList.Items {
-			key := rolesItem.Keys[0].Token.Text
+			key := rolesItem.Keys[0].Token.Value().(string)
```

Comment:
Is there ever a chance this might be `nil` (or not a string)? If so I'd prefer if you split this into two lines, something like 
```keyRaw := rolesItems.Keys[0].Token.Value()
key, ok := keyRaw.(string)
if !ok {
// Return some kind of error
}
```

Reply:
Done!